contracts/exchange/SmartWalletSwapImplementation.sol

```diff
@@ -332,10 +332,51 @@ contract SmartWalletSwapImplementation is SmartWalletSwapStorage, ISmartWalletSw
         );
     }
 
+    /// @dev borrow token from Lending platforms (AAVE, COMPOUND)
+    /// @param platform platform to borrow token
+    /// @param token underlying token to borrow, e.g ETH, USDT, DAI
+    /// @param borrowAmount amount of cToken (COMPOUND) or aToken (AAVE) to borrow
+    /// @param interestRateMode stable or variable rate mode
+    /// @param useGasToken whether to use gas token or not
+    function borrowFromLendingPlatform(
```

Comment:
We won't support borrowing from our contract, or here you want to support only AAVE v2?

Reply:
Yeah, we don't have to support in UI for now. But if ever we want to support borrowing, it will only be for Aave V2 and Compound.